test/support/iterator_utils.h

```diff
@@ -157,6 +157,9 @@ struct isReverse<::std::reverse_iterator<Iterator>> : ::std::true_type
 {
 };
 
+template <typename Iterator>
+inline constexpr bool is_reverse_v = isReverse<Iterator>::value;
```

Comment:
Can we rename `isReverse` to `is_reverse` in this PR to match style with others in this pattern and our normal style?

Reply:
Fixed.